src/QsCompiler/Tests.Compiler/SyntaxTests.fs

```diff
@@ -403,8 +403,8 @@ let ``Identifier tests`` () =
 [<Fact>]
 let ``Complex literal tests`` () =
     [
-        "[]", true, toExpr InvalidExpr, [ Error ErrorCode.EmptyValueArray ]
-        "[ ]", true, toExpr InvalidExpr, [ Error ErrorCode.EmptyValueArray ]
+        "[]", true, toArray [], []
```

Comment:
Are there some tests somewhere for the sized arrays?

Reply:
They are further down in the same `Complex literal tests` function.